### ISSUE TITLE
The plugin does not show up in a menu

### ISSUE THREAD

**Issue by LSergM** [esp8266/arduino-esp8266fs-plugin#99]
Setup:
Windows 10 Enterprise, version: 22H2
IDE: 2.2.1

It does work with IDE 1.8.19 with the same settings. (Look at the picture below)

![image](https://github.com/esp8266/arduino-esp8266fs-plugin/assets/26021027/1ed17af4-452c-45a5-bb00-0a29b17423a1)

![image](https://github.com/esp8266/arduino-esp8266fs-plugin/assets/26021027/f640d92f-6d3f-477e-a3dd-a60e42758051)

**Comment by Paschroeder** [comment]
Same for me with following Win Version and Arduino IDE 2.2.1:

Edition	Windows 11 Pro
Version	23H2
Installed on	‎07/‎05/‎2022
OS build	22631.2715
Experience	Windows Feature Experience Pack 1000.22677.1000.0

**Comment by candideu** [comment]
Ditto:

Arduino IDE 2.2.1
Windows 10
version 0.5.0 of plugin

**Comment by hobus1domini** [comment]
Hey guys!

Had the same problem. This is not part of this plugin, it's a general Arduino IDE 2 thing. Look here:
https://github.com/arduino/arduino-ide/issues/58

TL;DR: If u need tools / plugins, you are bound to the classic Arduino IDE for now! One would need a new plugin.

**Comment by candideu** [comment]
> Hey guys!
> 
> Had the same problem. This is not part of this plugin, it's a general Arduino IDE 2 thing. Look here: [arduino/arduino-ide#58](https://github.com/arduino/arduino-ide/issues/58)
> 
> TL;DR: If u need tools / plugins, you are bound to the classic Arduino IDE for now! One would need a new plugin.

I'm not sure if this will work for everyone's project, but I found a solution for mine using another plugin, and version 2.x of the IDE. Here's my breakdown: https://github.com/iffybooks/pocket-wifi-portal/issues/2#issuecomment-1837755761

**Comment by per1234** [comment]
If you are OK with using littlefs instead of the deprecated SPIFFS, an Arduino IDE 2.x filesystem uploader extension is available. See this: https://github.com/esp8266/arduino-esp8266fs-plugin/issues/96#issuecomment-1747831656
